### PR TITLE
Adapt the run-telepresence.sh script to work with the dev cluster too.

### PR DESCRIPTION
**Pull request by ableuler**

### CODE REVIEW

**Comment by vfried** [comment]
It seems to work for me

**Comment by ableuler** [comment]
@vfried please formally review/approve the PR, otherwise it can't be merged.

**Comment by lorenzo-cavazzi** [comment]
not working for me for other reasons. Go on and merge it, I'll create a separate issue

File: run-telepresence.sh [hunk 1]
```diff
@@ -18,27 +18,40 @@
...
+  K8S_NAMESPACE=renku
 else
   echo "You are going to exchange k8s deployments using the following context: ${CURRENT_CONTEXT}"
   read -p "Do you want to proceed? [y/n]"
   if [[ ! $REPLY =~ ^[Yy]$ ]]
   then
       exit 1
   fi
+
+  if [[ ! $K8S_NAMESPACE ]]
```
Comment (ciyer) [inline run-telepresence.sh]: Can we default K8S_NAMESPACE to $DEV_NAMESPACE which is used in the deployments instructions?

https://github.com/SwissDataScienceCenter/sdsc-dev-docs/tree/master/dev-deployments

Comment (ableuler) [reply run-telepresence.sh]: That wouldn't make much sense for the minikube case...

**Pull request merged**